sidechain/consensus/aura/src/verifier.rs

```diff
@@ -143,14 +143,20 @@ fn verify_block_ancestry<SidechainBlock: SidechainBlockTrait>(
 		last_block.block_number() + 1 == block.block_number(),
 		ConsensusError::BlockAncestryMismatch(
 			last_block.block_number(),
-			"Invalid block number".into()
+			last_block.hash(),
+			format!(
+				"Invalid block number, {} does not succeed {}",
+				block.block_number(),
+				last_block.block_number()
+			)
```

Comment:
The `BlockAncestryMismatch` now also contains the hash of the last known sidechain block. This is needed for fetching sidechain blocks from a peer.